config/locales/en.yml

```diff
@@ -341,6 +340,12 @@ en:
           nino: National Insurance number
           address: Correspondence address
           email: Email address
+    outstanding_mortgages:
+      show:
+        h1-heading: What is the outstanding mortgage?
+      field_set_header: What is the outstanding mortgage on your client's home?
+      outstanding_mortgage_amount: Enter the outstanding mortgage amount
+      hint: ' '
```

Comment:
Should there be some hint text here? If not, can the line be removed?

Reply:
I need to confirm with Simon so its a placeholder for now. I was also testing to see if this would insert a blank hint to give me some spacing😜

Reply:
👍